packages/console/src/pages/Connectors/components/ConnectorRow/index.tsx

```diff
@@ -7,31 +7,53 @@ import Status from '@/components/Status';
 import { connectorTitlePlaceHolder } from '@/consts/connectors';
 
 import ConnectorName from '../ConnectorName';
+import * as styles from './index.module.scss';
 
 type Props = {
   type: ConnectorType;
   connector?: ConnectorDTO;
+  connectors?: ConnectorDTO[];
```

Comment:
I don't think we should keep both `connector` and `connectors`.
It would be good to remove `connector` and keep `connectors` as required param.
You can allow connectors to be `ConnectorDTO | ConnectorDTO[]` and use `isArray` to check the input.
Or just simply make it `ConnectorDTO[]` and only allow an array.

Reply:
updated